src/main/java/com/sapred/ordermanagerred/repository/UserRepository.java

```diff
@@ -13,5 +14,9 @@ public interface UserRepository extends MongoRepository<User, String> {
     User getByAddressEmail(String email);
 
     @Autowired
-    List<User> getAllByCompanyId(String companyId);
+    User getById(String id);
```

Comment:
you can use findById, no need to implement it

Reply:
את הפונקציה הזו באמת התלבטתי מה באמת לעשות, בגלל שהפונקציה המובנית מחזירה רכיב Optional ואז צריך גם פונקציה get , אבל בכל אופן זה מעניין כי באמת עדיף ממש כמה שפחות לממש לבד.

Reply:
so please remove it